### PR TITLE
Remove Hazelcast address from node heartbeat table

### PR DESCRIPTION
**Pull request by a5anka**

### CODE REVIEW

File: components/org.wso2.carbon.business.messaging.core/src/main/java/org/wso2/carbon/business/messaging/core/internal/BrokerServiceComponent.java [hunk 1]
```diff
@@ -144,24 +135,11 @@ public void start(BundleContext context) throws AndesException {
                 AndesContext.getInstance().setClusteringEnabled(false);
```
Comment (Asitha) [inline components/org.wso2.carbon.business.messaging.core/src/main/java/org/wso2/carbon/business/messaging/core/internal/BrokerServiceComponent.java]: We still have a distinction between standalone and clustering?

File: .gitignore [hunk 2]
```diff
@@ -9,3 +9,6 @@ target
 *.ipr
 *.iws
 *.db
+
+# Maven generated resource files
+features/org.wso2.carbon.business.messaging.core.feature/src/main/resources
```
Comment (bsenduran) [inline .gitignore]: IMO, it is better to generate the source in the target directory

File: components/org.wso2.carbon.business.messaging.core/src/main/java/org/wso2/carbon/business/messaging/core/internal/BrokerServiceComponent.java (hunk 1 above)
Comment (a5anka) [reply components/org.wso2.carbon.business.messaging.core/src/main/java/org/wso2/carbon/business/messaging/core/internal/BrokerServiceComponent.java]: Yes. I did not remove it. Standalone mean non cluster aware.

File: .gitignore (hunk 2 above)
Comment (a5anka) [reply .gitignore]: Thanks @bsenduran. Will do in the later iteration. Tracked with https://github.com/wso2/carbon-business-messaging/issues/576

**Pull request merged**